pkg/cloudprovider/aws/instancetype.go

```diff
@@ -136,6 +129,8 @@ func (i *InstanceType) Overhead() v1.ResourceList {
 				// system-reserved
 				100+
 				// eviction threshold https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/pkg/kubelet/apis/config/v1beta1/defaults_linux.go#L23
+				100+
```

Comment:
I think this should be included in system-reserved according to k8s.
- kernel
- ssmd
- sshd
- other agents

Might be worth extracting a constant and adding a ton of text :D